ADALiOS/ADALiOS/ADALiOS.h

```diff
@@ -20,11 +20,11 @@
 //version in static define until we identify a better place:
 #define ADAL_VER_HIGH       2
 #define ADAL_VER_LOW        0
-#define ADAL_VER_PATCH      0
```

Comment:
I'm going to bump it up to 2.1 when we make a public release, for now I'm reserving 2.0.x for internal releases.